cclib/parser/nwchemparser.py

```diff
@@ -605,6 +620,12 @@ def extract(self, inputfile, line):
         if "Dispersion correction" in line:
             dispersion = utils.convertor(float(line.split()[-1]), "hartree", "eV")
             self.append_attribute("dispersionenergies", dispersion)
+            
+        # type of dispersion
+        if line.strip().find('disp vdw 3') > -1:

```

Comment:
We need a test for this because these lines aren't present at all in our examples.  From `dvb_dispersion_bp86_d3zero.out`,
```
      Dispersion Parameters
      ---------------------

             DFT-D3 Model
          s8 scale factor  :        1.000000000000
          sr6 scale factor :        1.683000000000
          sr8 scale factor :        1.139000000000
              vdW contrib  :        1.000000000000


             DFT-D3 Model
          s8 scale factor  :       -0.014719930985
          sr6 scale factor :
```
This is D3 (D3(zero), no damping), I haven't tried D3(BJ) in NWChem yet.

Reply:
I see the `disp vdw 3` in the input, but that is not repeated in the output from what I can tell.  This might need to be changed in general.  (mostly just a note to myself so i don't forget)

Reply:
https://nwchemgit.github.io/Density-Functional-Theory-for-Molecules.html?h=disp#disp-empirical-long-range-contribution-vdw

Reply:
I've added a file to test this for D3BJ with the dvb molecule, but it wouldn't be used through the typical dispersion tests on D3, but instead just to check metadata parsing. Is this overkill?

Reply:
Sorry I missed this, I think it's ok since more people are doing D3(BJ) than D3(0).